MODULE_NOT_FOUND on cdktf-cli@0.14.0

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### cdktf & Language Versions

- node version 16.18.1
- cdktf-cli version 0.14.0  (no problem on `0.13.3`)

### Affected Resource(s)

`cdktf-cli` is unusable

### Debug Output

https://gist.github.com/willish/e661ca7eccf7ade977e1d523b0fd76de

### Expected Behavior

`cdktf` should show the help

### Actual Behavior

A stacktrace due to `MODULE_NOT_FOUND` (`Error: Cannot find module '../../../../cdktf-cli/src/lib/version'`) in `constructs-maker.js`

### Steps to Reproduce

On Amazon Linux 2:

```
sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo
sudo yum -y install terraform
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
. ~/.nvm/nvm.sh
nvm install 16
npm install --global cdktf-cli@latest
cdktf help
```

--- a/packages/@cdktf/provider-generator/lib/get/constructs-maker.ts
+++ b/packages/@cdktf/provider-generator/lib/get/constructs-maker.ts
@@ -3,7 +3,7 @@
 import * as fs from "fs-extra";
 import * as path from "path";
 import { CodeMaker } from "codemaker";
-import { mkdtemp } from "../util";
+import { DISPLAY_VERSION, mkdtemp } from "../util";
 import * as srcmak from "jsii-srcmak";
 import {
   TerraformModuleConstraint,
@@ -19,7 +19,6 @@ import { TerraformProviderGenerator } from "./generator/provider-generator";
 import { ModuleGenerator } from "./generator/module-generator";
 import { ModuleSchema } from "./generator/module-schema";
 import { logTimespan } from "../config";
-import { DISPLAY_VERSION } from "../../../../cdktf-cli/src/lib/version";
 
 export enum Language {
   TYPESCRIPT = "typescript",

--- a/packages/@cdktf/provider-generator/lib/util.ts
+++ b/packages/@cdktf/provider-generator/lib/util.ts
@@ -96,3 +96,10 @@ export function uppercaseFirst(str: string): string {
   }
   return `${str[0].toLocaleUpperCase()}${str.slice(1)}`;
 }
+
+export const DISPLAY_VERSION = `${versionNumber()}`;
+
+function versionNumber(): string {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  return require("../package.json").version.replace(/\+[0-9a-f]+$/, "");
+}

